state/etcdstatedriver.go

```diff
@@ -53,14 +53,23 @@ type EtcdStateDriver struct {
 // Init the driver with a core.Config.
 func (d *EtcdStateDriver) Init(instInfo *core.InstanceInfo) error {
 	var err error
+	var endpoint *url.URL
 
-	if instInfo == nil || !strings.Contains(instInfo.DbURL, "etcd://") {
-		return errors.New("invalid etcd config")
+	if instInfo == nil || instInfo.DbURL == "" {
+		return core.Errorf("No Etcd config found")
```

Comment:
The error shouldn't start with a capital letter.

Reply:
done